members/federal_us_veterans_sec_denis_mcdonough.yaml

```diff
@@ -0,0 +1,206 @@
+          Colorado: Colorado
+          Connecticut: Connecticut
+          Delaware: Delaware
+          District of Columbia: District of Columbia
+          Florida: Florida
+          Georgia: Georgia
+          Hawaii: Hawaii
+          Idaho: Idaho
+          Illinois: Illinois
+          Indiana: Indiana
+          Iowa: Iowa
+          Kansas: Kansas
+          Kentucky: Kentucky
+          Louisiana: Louisiana
+          Maine: Maine
+          Manitoba: Manitoba
+          Maryland: Maryland
+          Massachusetts: Massachusetts
+          Michigan: Michigan
+          Minnesota: Minnesota
+          Mississippi: Mississippi
+          Missouri: Missouri
+          Montana: Montana
+          Nebraska: Nebraska
+          Nevada: Nevada
+          New Brunswick: New Brunswick
+          New Hampshire: New Hampshire
+          New Jersey: New Jersey
+          New Mexico: New Mexico
+          New York: New York
+          North Carolina: North Carolina
+          North Dakota: North Dakota
+          Nova Scotia: Nova Scotia
+          Ohio: Ohio
+          Oklahoma: Oklahoma
+          Ontario: Ontario
+          Oregon: Oregon
+          Pennsylvania: Pennsylvania
+          Philippines: Philippines
+          Prince Edward Island: Prince Edward Island
+          Puerto Rico: Puerto Rico
+          Quebec: Quebec
+          Rhode Island: Rhode Island
+          Saskatchewan: Saskatchewan
+          South Carolina: South Carolina
+          South Dakota: South Dakota
+          Tennessee: Tennessee
+          Texas: Texas
+          Utah: Utah
+          Vermont: Vermont
+          Virginia: Virginia
+          Washington: Washington
+          West Virginia: West Virginia
+          Wisconsin: Wisconsin
+          Wyoming: Wyoming
+          Yukon: Yukon
+          Other/Not in US: Other/Not in US
+    - fill_in:
+      - name: First Name
+        selector: input[id*='.first_name']
+        value: $NAME_FIRST
+        required: true
+      - name: Last Name
+        selector: input[id*='.last_name']
+        value: $NAME_LAST
+        required: true
+      - name: phone
+        selector: input[id*='.telephone_number']
+        value: $PHONE
+        required: false
+      - name: street
+        selector: input[id*='.street']
+        value: $ADDRESS_STREET
+        required: false
+      - name: city
+        selector: input[id*='.city']
+        value: $ADDRESS_CITY
+        required: false
+      - name: zip5
+        selector: input[id*='.zipcode']
+        value: $ADDRESS_ZIP5
+        required: true
+      - name: message #this is on purpose and is intended to bypass a modal that pops up when filling out the form
+        selector: textarea[id*='Threads']
+        value: "Attempting to workaround issues with contact form. If you see this message, please contact AdvocacySupport@everyaction.com"
+        required: true
+    - javascript:
+      - name: Verify email
+        values: [ "$EMAIL" ]
```

Comment:
this `values` is unused by the command, so you should just exclude it

Reply:
excluded it